core/src/main/java/hudson/security/AccessDeniedException2.java

```diff
@@ -12,6 +12,9 @@
  * @author Kohsuke Kawaguchi
  */
 public class AccessDeniedException2 extends AccessDeniedException {
+
+    private static final int MAX_REPORTED_AUTHORITIES = 10;
```

Comment:
Seems to be too low to be useful, especially given crazy behaviors such as https://github.com/jenkinsci/ldap-plugin/blob/4ced65b080ff3eb81ad23916ccb6133077b80185/src/main/java/hudson/security/LDAPSecurityRealm.java#L1039